java.lang.NoClassDefFoundError: scala/actors/threadpool/Arrays

Hello

To be honest i am not sure what i did. Only thing i remember is i clicked on some buttons xD This is using the newest Quark. Thought i would report it just in case (But may be just a derp)

---- Minecraft Crash Report ----

WARNING: coremods are present:
  AppleCore (AppleCore-mc1.10.2-2.0.1.jar)
  EnderCorePlugin (EnderCore-1.10.2-0.4.1.58-beta.jar)
  HFTransformer (Harvest-Festival-1.10.2-0.5.14.jar)
  IvToolkit (IvToolkit-1.2.8.5.jar)
  TANLoadingPlugin (ToughAsNails-1.9.4-1.0.11.jar)
  LoadingPlugin (ResourceLoader-MC1.9.4-1.5.1.jar)
  FMLPlugin (InventoryTweaks-1.61-58.jar)
  LoadingPlugin (Quark-r1.0-58.jar)
Contact their authors BEFORE contacting forge

// Everything's going to plan. No, really, that was supposed to happen.

Time: 11/6/16 3:30 PM
Description: Rendering screen

java.lang.NoClassDefFoundError: scala/actors/threadpool/Arrays
	at vazkii.quark.management.client.gui.GuiButtonChest.func_146112_a(GuiButtonChest.java:104)
	at net.minecraft.client.gui.GuiScreen.func_73863_a(GuiScreen.java:71)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:81)
	at net.minecraft.client.renderer.InventoryEffectRenderer.func_73863_a(InventoryEffectRenderer.java:51)
	at net.minecraft.client.gui.inventory.GuiInventory.func_73863_a(SourceFile:47)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:382)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1094)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1076)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:395)
	at org.multimc.EntryPoint.listen(EntryPoint.java:170)
	at org.multimc.EntryPoint.main(EntryPoint.java:54)
Caused by: java.lang.ClassNotFoundException: scala.actors.threadpool.Arrays
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 24 more
Caused by: java.lang.NullPointerException


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at vazkii.quark.management.client.gui.GuiButtonChest.func_146112_a(GuiButtonChest.java:104)
	at net.minecraft.client.gui.GuiScreen.func_73863_a(GuiScreen.java:71)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:81)
	at net.minecraft.client.renderer.InventoryEffectRenderer.func_73863_a(InventoryEffectRenderer.java:51)
	at net.minecraft.client.gui.inventory.GuiInventory.func_73863_a(SourceFile:47)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:382)

-- Screen render details --
Details:
	Screen name: net.minecraft.client.gui.inventory.GuiInventory
	Mouse location: Scaled: (292, 103). Absolute: (584, 272)
	Screen size: Scaled: (427, 240). Absolute: (854, 480). Scale factor of 2

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['kehaan'/267, l='MpServer', x=92.08, y=68.00, z=143.05]]
	Chunk stats: MultiplayerChunkCache: 623, 623
	Level seed: 0
	Level generator: ID 06 - BIOMESOP, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (88,64,132), Chunk: (at 8,4,4 in 5,8; contains blocks 80,0,128 to 95,255,143), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 17748 game time, 34155 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 165 total; [EntityGuardian['Guardian'/266247, l='MpServer', x=161.34, y=38.00, z=161.06], EntityItem['item.item.dyePowder.black'/198155, l='MpServer', x=38.51, y=32.00, z=174.45], EntityBlowfish['Blowfish'/115246, l='MpServer', x=103.70, y=60.08, z=140.06], EntityGuardian['Guardian'/270909, l='MpServer', x=62.75, y=11.00, z=215.42], EntityCow['Cow'/58, l='MpServer', x=74.71, y=65.00, z=109.81], EntityItem['item.item.dyePowder.black'/284730, l='MpServer', x=44.72, y=35.00, z=178.88], EntityGuardian['Guardian'/283718, l='MpServer', x=159.23, y=49.82, z=150.84], EntityBlowfish['Blowfish'/249408, l='MpServer', x=155.57, y=48.43, z=97.63], EntityGuardian['Guardian'/259655, l='MpServer', x=144.80, y=46.00, z=195.17], EntityBlowfish['Blowfish'/249419, l='MpServer', x=122.96, y=60.66, z=85.70], EntityGuardian['Guardian'/286797, l='MpServer', x=32.50, y=35.20, z=194.50], EntityCow['Cow'/74, l='MpServer', x=86.77, y=66.00, z=101.40], EntityCow['Cow'/75, l='MpServer', x=77.87, y=65.00, z=109.15], EntityBlowfish['Blowfish'/249416, l='MpServer', x=20.24, y=51.97, z=141.16], EntityEnderman['Enderman'/282184, l='MpServer', x=57.50, y=13.00, z=182.50], EntityNPCBuilder['Yulif'/77, l='MpServer', x=86.29, y=67.00, z=124.69], EntityDweller['Dweller'/282185, l='MpServer', x=54.50, y=13.00, z=184.50], EntityDweller['Dweller'/282186, l='MpServer', x=55.50, y=13.00, z=184.50], EntityGuardian['Guardian'/286794, l='MpServer', x=42.09, y=39.31, z=208.82], EntityPainting['Painting'/79, l='MpServer', x=94.50, y=69.50, z=117.03], EntityOctopus['Octopus'/257612, l='MpServer', x=161.92, y=62.52, z=85.47], EntityGuardian['Guardian'/286795, l='MpServer', x=39.81, y=44.04, z=187.38], EntityPainting['Painting'/80, l='MpServer', x=93.97, y=73.50, z=126.50], EntityCreeper['Creeper'/270420, l='MpServer', x=76.50, y=21.00, z=175.50], EntityPainting['Painting'/81, l='MpServer', x=93.97, y=73.50, z=124.50], EntityBat['Bat'/282709, l='MpServer', x=64.12, y=19.01, z=192.94], EntityItemFrame['entity.ItemFrame.name'/82, l='MpServer', x=93.03, y=73.50, z=119.50], EntityBat['Bat'/282710, l='MpServer', x=67.12, y=21.02, z=185.86], EntityItemFrame['entity.ItemFrame.name'/83, l='MpServer', x=93.03, y=73.50, z=117.50], EntityItemFrame['entity.ItemFrame.name'/84, l='MpServer', x=89.03, y=73.50, z=127.50], EntityItemFrame['entity.ItemFrame.name'/85, l='MpServer', x=89.03, y=73.50, z=123.50], EntityBlowfish['Blowfish'/249430, l='MpServer', x=96.17, y=61.86, z=163.46], EntityItemFrame['entity.ItemFrame.name'/86, l='MpServer', x=93.97, y=75.50, z=127.50], EntityBat['Bat'/282706, l='MpServer', x=67.25, y=20.42, z=179.77], EntityItemFrame['entity.ItemFrame.name'/87, l='MpServer', x=93.97, y=75.50, z=123.50], EntityOctopus['Octopus'/257620, l='MpServer', x=163.11, y=60.52, z=192.31], EntitySkeleton['Skeleton'/270419, l='MpServer', x=81.50, y=21.00, z=173.50], EntityBat['Bat'/282707, l='MpServer', x=76.44, y=20.00, z=181.51], EntityItemFrame['entity.ItemFrame.name'/88, l='MpServer', x=93.97, y=76.50, z=127.50], EntityItemFrame['entity.ItemFrame.name'/89, l='MpServer', x=93.97, y=76.50, z=125.50], EntityItemFrame['entity.ItemFrame.name'/90, l='MpServer', x=93.97, y=76.50, z=123.50], EntityNPCVillager['Abii'/91, l='MpServer', x=99.50, y=73.00, z=118.50], EntityItem['item.item.dyePowder.black'/197208, l='MpServer', x=69.30, y=34.00, z=176.61], EntityPainting['Painting'/92, l='MpServer', x=94.50, y=69.50, z=133.97], EntityZombie['Zombie'/268888, l='MpServer', x=135.50, y=20.00, z=206.50], EntityPainting['Painting'/93, l='MpServer', x=95.50, y=74.50, z=131.03], EntityZombie['Zombie'/268889, l='MpServer', x=144.29, y=22.00, z=205.50], EntityPainting['Painting'/94, l='MpServer', x=93.03, y=74.50, z=133.00], EntityBat['Bat'/278106, l='MpServer', x=108.25, y=21.10, z=128.46], EntityGuardian['Guardian'/282202, l='MpServer', x=166.85, y=48.00, z=143.41], EntityNPCVillager['Cloe'/95, l='MpServer', x=93.47, y=73.38, z=132.57], EntityOctopus['Octopus'/257628, l='MpServer', x=121.35, y=62.21, z=63.28], EntityNPCVillager['Mayor Jeimmi'/96, l='MpServer', x=99.55, y=69.00, z=126.28], EntityBlowfish['Blowfish'/249440, l='MpServer', x=112.07, y=61.53, z=70.88], EntitySquid['Squid'/246379, l='MpServer', x=104.25, y=61.12, z=93.29], EntityPainting['Painting'/112, l='MpServer', x=100.50, y=69.50, z=117.03], EntityItemFrame['entity.ItemFrame.name'/113, l='MpServer', x=97.50, y=69.50, z=119.97], EntityItemFrame['entity.ItemFrame.name'/114, l='MpServer', x=96.97, y=69.50, z=120.50], EntityItemFrame['entity.ItemFrame.name'/115, l='MpServer', x=101.97, y=73.50, z=118.50], EntityItem['item.item.dyePowder.black'/274551, l='MpServer', x=95.63, y=41.00, z=82.27], EntityPainting['Painting'/116, l='MpServer', x=101.00, y=74.50, z=119.97], EntityPainting['Painting'/117, l='MpServer', x=101.97, y=74.50, z=118.50], EntityCow['Cow'/119, l='MpServer', x=84.15, y=67.00, z=124.53], EntityOctopus['Octopus'/257652, l='MpServer', x=151.65, y=56.61, z=99.45], EntityGuardian['Guardian'/280691, l='MpServer', x=112.50, y=38.00, z=111.50], EntitySquid['Squid'/246395, l='MpServer', x=35.56, y=51.24, z=205.85], EntityAngler['Angler'/145016, l='MpServer', x=111.00, y=58.99, z=153.62], EntityPainting['Painting'/123, l='MpServer', x=100.50, y=69.50, z=133.97], EntityItemFrame['entity.ItemFrame.name'/124, l='MpServer', x=99.50, y=73.50, z=131.03], EntityPainting['Painting'/125, l='MpServer', x=99.50, y=74.50, z=131.03], EntityPainting['Painting'/126, l='MpServer', x=97.50, y=74.50, z=133.97], EntityOctopus['Octopus'/257661, l='MpServer', x=97.18, y=49.48, z=214.52], EntityZombie['Zombie'/276612, l='MpServer', x=72.50, y=15.00, z=145.78], EntityZombie['Zombie'/276614, l='MpServer', x=70.50, y=15.00, z=147.50], EntityAngler['Angler'/271495, l='MpServer', x=71.49, y=48.02, z=197.51], EntityZombie['Zombie'/276615, l='MpServer', x=70.50, y=15.00, z=146.50], EntityBat['Bat'/132, l='MpServer', x=101.52, y=41.35, z=146.46], EntityPig['Pig'/21639, l='MpServer', x=19.50, y=66.00, z=73.50], EntityPig['Pig'/21640, l='MpServer', x=19.50, y=66.00, z=72.50], EntityPig['Pig'/21641, l='MpServer', x=19.50, y=66.00, z=72.50], EntityAngler['Angler'/271502, l='MpServer', x=47.70, y=58.01, z=185.60], EntityBat['Bat'/266383, l='MpServer', x=96.51, y=47.46, z=145.46], EntitySquid['Squid'/246423, l='MpServer', x=58.03, y=61.67, z=125.03], EntityCreeper['Creeper'/254103, l='MpServer', x=128.50, y=16.00, z=143.50], EntityAngler['Angler'/271506, l='MpServer', x=119.18, y=55.01, z=112.65], EntityCreeper['Creeper'/279187, l='MpServer', x=131.84, y=13.00, z=188.63], EntityGuardian['Guardian'/281264, l='MpServer', x=34.65, y=38.18, z=163.03], EntityAngler['Angler'/271538, l='MpServer', x=32.61, y=57.07, z=160.55], EntitySkeleton['Skeleton'/197818, l='MpServer', x=96.50, y=39.00, z=153.29], EntitySkeleton['Skeleton'/197817, l='MpServer', x=98.54, y=39.00, z=152.22], EntityAngler['Angler'/271556, l='MpServer', x=152.61, y=54.02, z=125.38], EntityBat['Bat'/251585, l='MpServer', x=65.82, y=14.37, z=215.54], EntityItem['item.item.dyePowder.black'/285894, l='MpServer', x=105.92, y=44.00, z=96.26], EntityItem['item.item.dyePowder.black'/170198, l='MpServer', x=112.72, y=38.00, z=111.27], EntityCreeper['Creeper'/272607, l='MpServer', x=130.50, y=16.00, z=189.50], EntitySpider['Spider'/279263, l='MpServer', x=99.50, y=20.00, z=134.50], EntityGuardian['Guardian'/280792, l='MpServer', x=106.99, y=60.03, z=97.72], EntitySkeleton['Skeleton'/280802, l='MpServer', x=91.50, y=12.00, z=187.50], EntityGuardian['Guardian'/274156, l='MpServer', x=135.18, y=38.00, z=207.65], EntityGuardian['Guardian'/283890, l='MpServer', x=54.37, y=50.33, z=183.42], EntityItem['item.item.dyePowder.black'/221940, l='MpServer', x=47.16, y=35.00, z=147.83], EntityItem['item.item.dyePowder.black'/202497, l='MpServer', x=151.05, y=37.00, z=181.11], EntityBat['Bat'/282374, l='MpServer', x=119.52, y=22.37, z=198.02], EntityBat['Bat'/270092, l='MpServer', x=84.75, y=20.00, z=183.25], EntityPlayerSP['kehaan'/267, l='MpServer', x=92.08, y=68.00, z=143.05], EntityZombie['Zombie'/275208, l='MpServer', x=46.50, y=14.00, z=217.50], EntityZombie['Zombie'/275209, l='MpServer', x=43.50, y=14.00, z=222.50], EntityPainting['Painting'/92, l='MpServer', x=94.50, y=69.50, z=133.97], EntityPainting['Painting'/93, l='MpServer', x=95.50, y=74.50, z=131.03], EntityPainting['Painting'/94, l='MpServer', x=93.03, y=74.50, z=133.00], EntityItem['item.item.dyePowder.black'/278300, l='MpServer', x=77.55, y=33.00, z=206.13], EntityItem['item.item.dyePowder.black'/168729, l='MpServer', x=39.75, y=36.00, z=86.93], EntityGuardian['Guardian'/271653, l='MpServer', x=124.01, y=38.00, z=75.15], EntityPainting['Painting'/123, l='MpServer', x=100.50, y=69.50, z=133.97], EntityItem['item.item.dyePowder.black'/261413, l='MpServer', x=127.50, y=38.00, z=194.56], EntityPainting['Painting'/125, l='MpServer', x=99.50, y=74.50, z=131.03], EntityPainting['Painting'/126, l='MpServer', x=97.50, y=74.50, z=133.97], EntitySkeleton['Skeleton'/274733, l='MpServer', x=110.50, y=18.00, z=130.50], EntityPainting['Painting'/79, l='MpServer', x=94.50, y=69.50, z=117.03], EntityPainting['Painting'/80, l='MpServer', x=93.97, y=73.50, z=126.50], EntityPainting['Painting'/81, l='MpServer', x=93.97, y=73.50, z=124.50], EntitySquid['Squid'/246583, l='MpServer', x=94.77, y=48.64, z=87.85], EntityBat['Bat'/218425, l='MpServer', x=104.58, y=41.28, z=148.47], EntityBat['Bat'/218424, l='MpServer', x=104.51, y=41.00, z=146.49], EntityItem['item.item.dyePowder.black'/174911, l='MpServer', x=36.88, y=37.00, z=85.73], EntityItem['item.item.dyePowder.black'/231230, l='MpServer', x=89.88, y=34.00, z=177.68], EntityPainting['Painting'/112, l='MpServer', x=100.50, y=69.50, z=117.03], EntityPainting['Painting'/116, l='MpServer', x=101.00, y=74.50, z=119.97], EntityItem['item.item.dyePowder.black'/229184, l='MpServer', x=75.65, y=32.00, z=188.47], EntityPainting['Painting'/117, l='MpServer', x=101.97, y=74.50, z=118.50], EntitySkeleton['Skeleton'/278878, l='MpServer', x=66.50, y=37.00, z=135.50], EntityItem['item.item.dyePowder.black'/197981, l='MpServer', x=154.01, y=36.00, z=160.66], EntityItem['item.item.dyePowder.black'/271221, l='MpServer', x=138.16, y=38.00, z=81.35], EntityItem['item.item.dyePowder.black'/164215, l='MpServer', x=65.57, y=33.00, z=189.96], EntityItem['item.item.dyePowder.black'/275324, l='MpServer', x=71.95, y=37.00, z=170.38], EntityGuardian['Guardian'/265594, l='MpServer', x=68.50, y=38.40, z=174.65], EntityGuardian['Guardian'/267642, l='MpServer', x=48.40, y=34.00, z=219.08], EntitySpider['Spider'/230277, l='MpServer', x=131.91, y=15.00, z=140.91], EntityItem['item.item.dyePowder.black'/263043, l='MpServer', x=33.15, y=35.00, z=110.93], EntityItemFrame['entity.ItemFrame.name'/22933, l='MpServer', x=83.50, y=68.50, z=143.03], EntityItemFrame['entity.ItemFrame.name'/22934, l='MpServer', x=83.50, y=69.50, z=150.97], EntityItemFrame['entity.ItemFrame.name'/22936, l='MpServer', x=84.50, y=69.50, z=150.97], EntityItemFrame['entity.ItemFrame.name'/22937, l='MpServer', x=83.50, y=70.50, z=145.97], EntityItemFrame['entity.ItemFrame.name'/22938, l='MpServer', x=83.50, y=70.50, z=147.03], EntityGuardian['Guardian'/282014, l='MpServer', x=125.50, y=36.00, z=69.50], EntityItemFrame['entity.ItemFrame.name'/22939, l='MpServer', x=84.50, y=70.50, z=145.97], EntityItemFrame['entity.ItemFrame.name'/22940, l='MpServer', x=84.50, y=70.50, z=147.03], EntityNPCShopkeeper['Danieru'/22941, l='MpServer', x=83.50, y=68.00, z=148.50], EntityItem['item.item.dyePowder.black'/277411, l='MpServer', x=138.26, y=36.00, z=140.50], EntityItem['item.item.dyePowder.black'/288181, l='MpServer', x=103.70, y=55.71, z=92.77], EntityCreeper['Creeper'/260021, l='MpServer', x=79.50, y=17.00, z=140.50], EntityGuardian['Guardian'/230842, l='MpServer', x=91.50, y=36.00, z=208.50], EntityGuardian['Guardian'/274361, l='MpServer', x=121.16, y=35.00, z=141.86], EntityAngler['Angler'/240573, l='MpServer', x=81.50, y=55.93, z=168.33], EntitySkeleton['Skeleton'/275908, l='MpServer', x=61.50, y=36.00, z=130.50], EntitySkeleton['Skeleton'/275907, l='MpServer', x=60.50, y=36.00, z=131.50], EntityGuardian['Guardian'/285128, l='MpServer', x=31.50, y=42.70, z=182.50], EntityZombie['Zombie'/270807, l='MpServer', x=102.50, y=19.00, z=124.50], EntityItem['item.item.dyePowder.black'/263135, l='MpServer', x=108.77, y=42.00, z=162.88], EntityItem['item.item.dyePowder.black'/275423, l='MpServer', x=124.81, y=35.00, z=87.07], EntityBat['Bat'/274412, l='MpServer', x=54.89, y=36.05, z=125.59], EntityBat['Bat'/274413, l='MpServer', x=68.11, y=39.10, z=135.63], EntityGuardian['Guardian'/284141, l='MpServer', x=118.73, y=48.79, z=202.88], EntityItem['item.item.dyePowder.black'/271343, l='MpServer', x=63.88, y=57.00, z=160.59], EntityGuardian['Guardian'/285162, l='MpServer', x=15.58, y=46.52, z=190.42]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:415)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2660)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:392)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:395)
	at org.multimc.EntryPoint.listen(EntryPoint.java:170)
	at org.multimc.EntryPoint.main(EntryPoint.java:54)

-- System Details --
Details:
	Minecraft Version: 1.10.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_73, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1312996328 bytes (1252 MB) / 3093299200 bytes (2950 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx4096m
	IntCache: cache: 0, tcache: 0, allocated: 3, tallocated: 104
	FML: MCP 9.32 Powered by Forge 12.18.2.2106 48 mods loaded, 48 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.10.2-12.18.2.2106-universal.jar) 
	UCHIJAAAA	Forge{12.18.2.2106} [Minecraft Forge] (forge-1.10.2-12.18.2.2106-universal.jar) 
	UCHIJAAAA	ivtoolkit{1.2.8.5} [IvToolkit] (minecraft.jar) 
	UCHIJAAAA	immersiveengineering{0.10-48} [Immersive Engineering] (ImmersiveEngineering-0.10-48.jar) 
	UCHIJAAAA	JEI{3.12.8.325} [Just Enough Items] (jei_1.10.2-3.12.8.325.jar) 
	UCHIJAAAA	AppleCore{2.0.1} [AppleCore] (AppleCore-mc1.10.2-2.0.1.jar) 
	UCHIJAAAA	appleskin{1.0.4} [AppleSkin] (AppleSkin-mc1.10.2-1.0.4.jar) 
	UCHIJAAAA	aquaculture{1.4.0} [Aquaculture] (Aquaculture-1.10.2-1.4.0.28.jar) 
	UCHIJAAAA	Psi{r1.0-41} [Psi] (Psi-r1.0-41.jar) 
	UCHIJAAAA	Quark{r1.0-58} [Quark] (Quark-r1.0-58.jar) 
	UCHIJAAAA	AutoRegLib{1.0-2} [AutoRegLib] (AutoRegLib-1.0-2.jar) 
	UCHIJAAAA	Baubles{1.3.2} [Baubles] (Baubles-1.10.2-1.3.2.jar) 
	UCHIJAAAA	bettertitlescreen{1.2e} [Better Title Screen] (BetterTitleScreen-1.10.2-1.2e.jar) 
	UCHIJAAAA	BiomesOPlenty{5.0.0.2091} [Biomes O' Plenty] (BiomesOPlenty-1.10.2-5.0.0.2091-universal.jar) 
	UCHIJAAAA	Botania{r1.9-337} [Botania] (Botania r1.9-337.jar) 
	UCHIJAAAA	buttons{1.10.2-0.0.3-3} [Buttons] (Buttons-1.10.2-0.0.3-3.jar) 
	UCHIJAAAA	Chameleon{1.10-2.1.7} [Chameleon] (Chameleon-1.10-2.1.7.jar) 
	UCHIJAAAA	chancecubes{1.10.2-2.3.2.131} [Chance Cubes] (ChanceCubes-1.10.2-2.3.2.131.jar) 
	UCHIJAAAA	mcmultipart{1.3.0} [MCMultiPart] (MCMultiPart-1.3.0-universal.jar) 
	UCHIJAAAA	chiselsandbits{12.7} [Chisels & Bits] (chiselsandbits-12.7.jar) 
	UCHIJAAAA	MineTweaker3{3.0.13} [MineTweaker 3] (CraftTweaker-1.10.2-3.0.13.jar) 
	UCHIJAAAA	ctgui{1.0.0} [CT-GUI] (CraftTweaker-1.10.2-3.0.13.jar) 
	UCHIJAAAA	CustomMainMenu{2.0.2} [Custom Main Menu] (CustomMainMenu-MC1.10.2-2.0.2.jar) 
	UCHIJAAAA	dailies{1.10.2-6} [Bailey's Dailies] (dailies-1.10.2-6.jar) 
	UCHIJAAAA	enchiridion{3.1.12} [Enchiridion] (Enchiridion-1.10.2-3.1.12.jar) 
	UCHIJAAAA	endercore{1.10.2-0.4.1.58-beta} [EnderCore] (EnderCore-1.10.2-0.4.1.58-beta.jar) 
	UCHIJAAAA	FastLeaveDecay{1.1} [Fast Leave Decay] (FastLeaveDecay-MC1.10.2-1.2.2.jar) 
	UCHIJAAAA	forestry{5.2.16.315} [Forestry] (forestry_1.10.2-5.2.16.315.jar) 
	UCHIJAAAA	harvestfestival{0.5.14} [Harvest Festival] (Harvest-Festival-1.10.2-0.5.14.jar) 
	UCHIJAAAA	mtlib{@VERSION@} [MTLib] (MTLib-1.0.0.jar) 
	UCHIJAAAA	initialinventory{1.0.0} [InitialInventory] (InitialInventory-1.0.0.jar) 
	UCHIJAAAA	inventorytweaks{1.61-58-a1fd884} [Inventory Tweaks] (InventoryTweaks-1.61-58.jar) 
	UCHIJAAAA	journeymap{1.10.2-5.3.0b2} [JourneyMap] (journeymap-1.10.2-5.3.0b2-unlimited.jar) 
	UCHIJAAAA	mantle{1.10.2-1.1.1.194} [Mantle] (Mantle-1.10.2-1.1.1.jar) 
	UCHIJAAAA	mousetweaks{2.6.4} [Mouse Tweaks] (MouseTweaks-2.6.4-mc1.10.2.jar) 
	UCHIJAAAA	natura{1.10.2-4.0.0.87} [Natura] (natura-1.10.2-4.0.0.87.jar) 
	UCHIJAAAA	qcreatures{2.0.1} [Quintessential Creatures] (qcreatures-2.0.1.jar) 
	UCHIJAAAA	railcraft{10.0.0} [Railcraft] (Railcraft_1.10.2-10.0.0.jar) 
	UCHIJAAAA	reccomplex{1.2.3.4} [Recurrent Complex] (RecurrentComplex-1.2.3.4.jar) 
	UCHIJAAAA	ResourceLoader{1.5.1} [Resource Loader] (ResourceLoader-MC1.9.4-1.5.1.jar) 
	UCHIJAAAA	SimpleAchievements{MC1.9.4-1.2.2-26} [Simple Achievements] (SimpleAchievements-MC1.9.4-1.2.2-26.jar) 
	UCHIJAAAA	soulshardstow{1.10.2-2.6.5-45} [Soul Shards - The Old Ways] (SoulShards-TOW-1.10.2-2.6.5-45.jar) 
	UCHIJAAAA	SpiceOfLife{1.3.6} [The Spice of Life] (SpiceOfLife-mc1.9.4-1.3.6.jar) 
	UCHIJAAAA	StorageDrawers{1.10.2-3.4.2} [Storage Drawers] (StorageDrawers-1.10.2-3.4.2.jar) 
	UCHIJAAAA	rscircuits{0.0.1} [Super Circuit Maker] (SuperCircuitMaker-1.0.1.jar) 
	UCHIJAAAA	ToughAsNails{1.0.11} [Tough As Nails] (ToughAsNails-1.9.4-1.0.11.jar) 
	UCHIJAAAA	waddles{0.5.0} [Waddles] (Waddles-1.10.2-0.5.0.jar) 
	Loaded coremods (and transformers): 
AppleCore (AppleCore-mc1.10.2-2.0.1.jar)
  squeek.applecore.asm.TransformerModuleHandler
EnderCorePlugin (EnderCore-1.10.2-0.4.1.58-beta.jar)
  com.enderio.core.common.transform.EnderCoreTransformer
HFTransformer (Harvest-Festival-1.10.2-0.5.14.jar)
  joshie.harvest.asm.HFTransformer
IvToolkit (IvToolkit-1.2.8.5.jar)
  
TANLoadingPlugin (ToughAsNails-1.9.4-1.0.11.jar)
  toughasnails.asm.transformer.BlockCropsTransformer
  toughasnails.asm.transformer.EntityRendererTransformer
  toughasnails.asm.transformer.WorldTransformer
LoadingPlugin (ResourceLoader-MC1.9.4-1.5.1.jar)
  lumien.resourceloader.asm.ClassTransformer
FMLPlugin (InventoryTweaks-1.61-58.jar)
  invtweaks.forge.asm.ContainerTransformer
LoadingPlugin (Quark-r1.0-58.jar)
  vazkii.quark.base.asm.ClassTransformer
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 375.70' Renderer: 'GeForce GTX 980 Ti/PCIe/SSE2'
	Pulsar/natura loaded Pulses: 
		- NaturaCommons (Enabled/Forced)
		- NaturaOverworld (Enabled/Not Forced)
		- NaturaNether (Enabled/Not Forced)
		- NaturaEntities (Enabled/Not Forced)
		- NaturaOredict (Enabled/Forced)
		- NaturaWorld (Enabled/Not Forced)

	Launched Version: MultiMC5
	LWJGL: 2.9.1
	OpenGL: GeForce GTX 980 Ti/PCIe/SSE2 GL version 4.5.0 NVIDIA 375.70, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 8x Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz

--- a/src/main/java/vazkii/quark/management/client/gui/GuiButtonChest.java
+++ b/src/main/java/vazkii/quark/management/client/gui/GuiButtonChest.java
@@ -19,7 +19,7 @@ import net.minecraft.client.gui.GuiScreen;
 import net.minecraft.client.renderer.GlStateManager;
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.util.text.translation.I18n;
-import scala.actors.threadpool.Arrays;
+import java.util.Arrays;
 import vazkii.arl.util.RenderHelper;
 import vazkii.quark.management.feature.FavoriteItems;
 import vazkii.quark.management.feature.StoreToChests;

